initializeTailwindcssWorker.d.ts

```diff
@@ -0,0 +1,10 @@
+import { TailwindConfig } from 'tailwindcss/tailwind-config';
+
+export interface TailwindWorkerOptions {
+  /**
+   * Hook that will run before the tailwind config is used.
+   */
+  prepareTailwindConfig?: (tailwindConfig?: TailwindConfig) => TailwindConfig;
```

Comment:
it would propably better to type `tailwindConfig` as unknown, as we can allow passing a simple string and preparing it here to a `TailwindConfig` like play tailwind does it.

this way `tailwindConfig.plugins: string[]` would also work.

Reply:
This seems to imply you’re considering to use `eval`, which is something want to avoid.

The type can be

```ts
interface MonacoTailwindConfig extends Omit<TailwindConfig, 'plugins'> {
  plugins?: string[]
}
```

Reply:
wow nice one with Omit ^^
lets say - one want to use it like play.tailwind one has no chance other than eval. - and eval is fine at that point imo.
point is that one cannot use eval before passing this to the worker but one needs to use eval later in a custom `prepareTailwindConfig`

and this does work currently - as ts types are all fictionary and `tailwindConfig` can be a string. i just wanted to make this behaviour api.

Reply:
I have been trying to figure out how to implement this in a real world project and I now see the problem. Unfortunately that project will have to use `eval()` as well. :frowning_face: 

I think it’s nice to support passing the tailwind config either as an object or as a string, in which case it’s up the a custom worker implementation to resolve it.

Reply:
Typing this parameter also as string would require some more changes in other parts. I will address this in a second pr.